src/Dodo.HttpClient.ResiliencePolicies/RetryPolicy/RetryPolicySettings.cs

```diff
@@ -20,19 +41,26 @@ public RetryPolicySettings()
 				TimeSpan.FromMilliseconds(Defaults.Retry.MedianFirstRetryDelayInMilliseconds));
 
 			OnRetry = DoNothingOnRetry;
+			_retryCount = Defaults.Retry.RetryCount;
 		}
 
 		private RetryPolicySettings(
-			IEnumerable<TimeSpan> sleepDurationProvider)
+			int retryCount,
+			Func<int, DelegateResult<HttpResponseMessage>, Context, TimeSpan> sleepDurationProvider)
 		{
-			if (sleepDurationProvider == null)
-				throw new ArgumentNullException(nameof(sleepDurationProvider));
-
 			_sleepDurationProvider = sleepDurationProvider;
 			OnRetry = DoNothingOnRetry;
+			_retryCount = retryCount;
 		}
 
-		private static readonly Action<DelegateResult<HttpResponseMessage>, TimeSpan> DoNothingOnRetry = (_, __) => { };
+		private bool _useRetryAfter;
```

Comment:
Could we totally remove this along with EnableRetryAfterFeature and support this behaviour by default? Actually, there are no reasons to ignore RetryAfter header.

Reply:
Agree. I've remore and enabled RetryAfter by default.